apps/coscrad-frontend/src/components/resources/spatial-features/thumbnail-presenters/spatial-feature-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,54 @@
+import { GeometricFeatureType, ISpatialFeatureViewModel } from '@coscrad/api-interfaces';
+import { Card, CardContent, CardHeader, Divider } from '@mui/material';
+import { FunctionalComponent } from '../../../../utils/types/functional-component';
+import { LinePresenter } from './line-presenter';
+import { PointPresenter } from './point-presenter';
+import { PolygonPresenter } from './polygon-presenter';
+
+interface HasCoordinates<T = unknown> {
+    coordinates: T;
+}
+
+const lookupTable: { [K in GeometricFeatureType]: FunctionalComponent<HasCoordinates> } = {
+    [GeometricFeatureType.point]: PointPresenter,
+    [GeometricFeatureType.line]: LinePresenter,
+    [GeometricFeatureType.polygon]: PolygonPresenter,
+};
+
+export const SpatialFeatureDetailThumbnailPresenter = (
```

Comment:
This is where we could have a thumbnail map showing the location.  I wonder if there's a way to dynamically create a png out of a map view...

Reply:
That's actually achievable by reusing the `CoscradMap` with just a single spatial feature. We might have to find a way to lock scrolling and set the zoom, though.